packages/remix/src/utils/integrations/opentelemetry.ts

```diff
@@ -0,0 +1,24 @@
+import { RemixInstrumentation } from 'opentelemetry-instrumentation-remix';
+
+import { defineIntegration } from '@sentry/core';
+import { addOpenTelemetryInstrumentation } from '@sentry/node';
+import type { IntegrationFn } from '@sentry/types';
+import type { RemixOptions } from '../remixOptions';
+
+const _remixIntegration = ((options?: RemixOptions) => {
+  return {
+    name: 'Remix',
+    setupOnce() {
+      addOpenTelemetryInstrumentation(
+        new RemixInstrumentation({
+          actionFormDataAttributes: options?.sendDefaultPii ? options?.captureActionFormDataKeys : undefined,
+        }),
```

Comment:
m: We'll need to ensure we have correct origin & op for spans emitted here. Ideally we have some hook, or alternatively we could possibly extend the class, or something like this 🤔 See e.g. the express or fastify integration for how we do that in other places!

Reply:
Done 👍